subsys/bluetooth/host/att.c

```diff
@@ -3768,10 +3769,12 @@ int bt_eatt_reconfigure(struct bt_conn *conn, uint16_t mtu)
 	}
 
 	while (offset < i) {
+		mps = MIN(mtu + BT_L2CAP_SDU_HDR_SIZE, BT_L2CAP_RX_MTU);
```

Comment:
this could be done by stack if user pass mps=0 ?

Reply:
Out of scope of this PR.